### ISSUE TITLE
Latest version erroring out on import with PyPy

### ISSUE THREAD

**Issue by alex** [phihag/ipaddress#19]
The error message is:

```pytb
Traceback (most recent call last):
  File "<builtin>/app_main.py", line 75, in run_toplevel
  File "<builtin>/app_main.py", line 601, in run_it
  File "<string>", line 1, in <module>
  File "/Users/alex_gaynor/.virtualenvs/tempenv-24da2037275be/site-packages/ipaddress.py", line 1358, in <module>
    class IPv4Address(_BaseV4, _BaseAddress):
TypeError: __weakref__ slot disallowed: we already got one
```

I believe putting `__slots__ = ()` on `_TotalOrderingMixin` will fix it (but I haven't tested)

**Comment by phihag** [comment]
Thank you for the report. I believe this to be fixed in ipaddress version 1.0.10.

**Comment by alex** [comment]
Awesome, thank you!
